Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -45,6 +45,187 @@ COREGRAPHICS_EXPORT void CGImageAddDestructionListener(CGImageDestructionListene
 
 #pragma endregion OLD_CODE
```

Comment:
that's pretty much the old code. given compositor uses this to release the textured images, we need a support for this.
After the refactor it can be simplified. 

---
In reply to: [88774081](https://github.com/Microsoft/WinObjC/pull/1430#discussion_r88774081) [](ancestors = 88774081)